lib/koala/api/graph_batch_api.rb

```diff
@@ -8,6 +8,11 @@ class GraphBatchAPI < API
       # inside a batch call we can do anything a regular Graph API can do
       include GraphAPIMethods
 
+      # Limits from @see https://developers.facebook.com/docs/marketing-api/batch-requests/v2.8
+      MAX_AD_SUBMISSIONS = 10
```

Comment:
From what I can tell, altho FB is hard about `MAX_CALLS`, `MAX_AD_SUBMISSIONS` may be just a recommendation.